components/service/experiments/src/test/java/mozilla/components/service/experiments/ActiveExperimentTest.kt

```diff
@@ -97,9 +99,27 @@ class ActiveExperimentTest {
         active = ActiveExperiment.load(context, experiments)
         assertEquals(experiments[1], active!!.experiment)
         assertEquals("branch3", active.branch)
+    }
+
+    @Test
+    fun `test load invalid experminet from storage`() {
```

Comment:
nit:
```suggestion
    fun `test load invalid experiment from storage`() {
```

Reply:
Oops. I think uncle Sigmund can be proud of me. :sweat_smile: 

Thanks!